client/state/wordads/approve/reducer.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	WORDADS_SITE_APPROVE_REQUEST,
+	WORDADS_SITE_APPROVE_REQUEST_SUCCESS,
+	WORDADS_SITE_APPROVE_REQUEST_FAILURE,
+	WORDADS_SITE_APPROVE_REQUEST_DISMISS_ERROR,
+	SERIALIZE,
+	DESERIALIZE
+} from 'state/action-types';
+
+/**
+ * Tracks all known user objects, indexed by user ID.
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action payload
+ * @return {Object}        Updated state
+ */
+
+const initialState = {
+	requesting: false,
+	error: null
+};
+
+export function requestStatus( state = initialState, action ) {
```

Comment:
Are word ad requests per user, or per site? If it's the latter I'd suggest indexing by site id, or clearing state when switching sites.